### DISCUSSION TITLE
server monetization JavaScript  advertising

### DISCUSSION THREAD

**Discussion by DarkShyMW** [misskey-dev/misskey#9496] [General]
Good time. I am using the latest build Misskey. I saw this message that you need sponsorship. but who will help the server administrators? as for me - implement JavaScript advertising. it will help us and you.
![Screenshot_2023-01-08-21-05-54-458-edit_com android chrome](https://user-images.githubusercontent.com/30442729/211211950-79ebca26-012e-43a8-b1b5-fd7b6a930ebb.jpg)

**Comment by DarkShyMW** [comment]
May Be you Haven some information about this situation?

**Reply by tassoman** [reply]
Ads functionality it's already in, since v12.x.

**Reply by DarkShyMW** [reply]
There is no way to insert JavaScript

**Comment by mishii09** [comment]
Hi DarkShyMW,

Implementing JavaScript advertising could indeed help with server monetization. It’s a practical solution, similar to choosing the best route from [Leicester to London](https://britroute.co.uk/coach-hire/)—both offer a way to streamline processes and achieve goals effectively. Hope this helps!
